src/main/resources/com/cloudbees/plugins/credentials/CredentialsStoreAction/CredentialsWrapper/sidepanel.jelly

```diff
@@ -27,10 +27,10 @@
   <l:side-panel>
     <l:tasks>
     <j:set var="url" value="${h.getNearestAncestorUrl(request,it)}"/>
-      <l:task icon="images/24x24/up.png" href="../.." title="${%backto(it.domain.displayName)}" contextMenu="false"/>
-      <l:task icon="images/24x24/setting.png" href="${url}/update" title="${%Update}" permission="${it.domain.parent.UPDATE}"/>
-      <l:task icon="images/24x24/edit-delete.png" href="${url}/delete" title="${%Delete}" permission="${it.domain.parent.DELETE}"/>
-      <l:task icon="plugin/credentials/images/24x24/move.png" href="${url}/move" title="${%Move}" permission="${it.domain.parent.DELETE}"/>
+      <l:task icon="images/svgs/up.svg" href="../.." title="${%backto(it.domain.displayName)}" contextMenu="false"/>
```

Comment:
So IIUC this will not work with the current `jenkins.version` in `pom.xml`?

Reply:
The SVG icons are only available on versions equal or higher than 2.308. I just pushed a change that addresses that.